SPECS/shim-unsigned-aarch64/shim-unsigned-aarch64.signatures.json

```diff
@@ -1,7 +1,9 @@
 {
  "Signatures": {
-  "cbl-mariner-ca-20210127.der": "34f7436bc56c7b4e526e55dbc207ad6311dbace0403e1d462ad1f9ce3479d386",
-  "sbat.csv.in": "895ecdff532c9ed1cfbcd1e4d8fd9b29b87628394cc9c9101ec5be47f66e504d",
-  "shim-15.4.tar.bz2": "8344473dd10569588b8238a4656b8fab226714eea9f5363f8c410aa8a5090297"
+  "azurelinux-ca-20230216.der": "fac39cd3bbb87d6326eb87020eb7ab1f01058d9b94d3c47d4264da05601bc89a",
+  "sbat.azurelinux.csv": "bb37f33ec9bb6be2fa8929243b613337244f007efb0761196f2f6b3caa03926c",
+  "shim-15.8.tar.bz2": "a79f0a9b89f3681ab384865b1a46ab3f79d88b11b4ca59aa040ab03fffae80a9",
+  "shim-find-debuginfo.sh": "80a43905c7ec5578794adcbf277e01a6d7eb05f2c12bd97bddc8d727066266df",
```

Comment:
> shim-find-debuginfo.sh 's hash is invalid, needs to updated to match the signature in shim-unsigned-x64.

thanks, updated

> 
> Sidenote: I think we're running into a similar issue we were trying to avoid with the symlink where we have the same dependencies between x86_64 and aarch64

If you mean separate src packages for each shim-unsigned arch, then I agree they could be combined into one, but the difficulty then is if one arch needs to be updated to a newer version while the other arch can't, maybe due to a arch-specific bug. Fedora handles this with separate src packages for shim-unsigned and also having a 'map' of what shim-unsigned package version is used by the actual shim package. I don't know if that's the best way to handle it, but in any case I think it will require more thinking/designing to come up with the best way to arrange the packages for both/all archs, which probably is best done for the next version instead of 3.0.